### ISSUE TITLE
FDM Issues

### ISSUE THREAD

**Issue by ghost** [viktorradnai/flightgear-ask21#132]
This is a collection issue for problems with the current FDM
1. Stall behaviour: This might be a problem of the FDM file or of YASim: When flying close to stall point, the real aircraft descend very fast while keeping the nose open (called 'Sackflug' in german), this is not working in FG, there it's just descending a tiny bit faster as usually with ~1m/s:
![fgfs-screen-263](https://cloud.githubusercontent.com/assets/12275960/25175258/6a82c5fe-24fa-11e7-979c-73a5f8da9669.png)
2. Spin: Is not possible at all with our flightgear version as it gets into a kind of spiral but never seems to have a one-wing-stall, IRL only possible if pilot+copilot are light or with spin ballast attached.
Besides that, very very well done @viktorradnai  :D with the new version

**Comment by viktorradnai** [comment]
@D-ECHO the behaviour you describe with 1. is called mushing in English. Unfortunately I haven't figured out whether it's possible to simulate at all with YASim. The same goes for spinning, 

I know that YASim does not simulate stalled conditions well, it's only meant to be accurate during normal flight, so I'm not sure if this can be fixed. I was considering rewriting the FDM using JSBSim but it looked highly complex. 

Can you think of a YASim aircraft with good mush / stall / spin behaviour? If you give me a working example I might be able to learn some tricks from it.

**Comment by viktorradnai** [comment]
The suggestion by the Flightgear devs was to rewrite the FDM in JSBSim which is a ton of work but may be necessary eventually.

**Comment by ghost** [comment]
@viktorradnai Yep, that's probably the only perspective on a long-term look, but I myself were not able yet to do so

**Comment by viktorradnai** [comment]
I haven't got the time to start on this, maybe around Christmas if things get a bit more quiet. Should we close this issue?

**Comment by ghost** [comment]
I'd say keep it open, it doesn't bother and this way we don't forget it

**Comment by ghost** [comment]
probably will be fixed by #136

**Comment by ghost** [comment]
Didn't want to close it :D
